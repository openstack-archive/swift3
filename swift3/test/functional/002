#!/bin/bash

# PUT Bucket

. ./common

echo '
<CreateBucketConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
<LocationConstraint>US</LocationConstraint>
</CreateBucketConfiguration>' > $tmp.default.xml

S3USER=tester

_s3_put /bucket -D - -T $tmp.default.xml | _filter_curl
