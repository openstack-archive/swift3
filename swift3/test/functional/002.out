QA output created by 002
> s3curl --id tester -- -X PUT -D - -T /TMP.default.xml http://SWIFT_HOST/bucket... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
Location: /bucket
X-Trans-Id: TXID

